buildenv/jenkins/variables/defaults.yml

```diff
@@ -395,12 +395,16 @@ x86-64_linux:
   node_labels:
     build:
       all: 'ci.role.build && hw.arch.x86 && sw.os.cent.7'
-      8: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
-      11: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
+      8: 'ci.role.build && hw.arch.x86 && sw.tool.docker'
+      11: 'ci.role.build && hw.arch.x86 && sw.tool.docker'
+    docker_image:
+      all: 'eclipseopenj9/jenkins-agent-x86-centos6:13'
   extra_configure_options:
     all: '--enable-jitserver'
   build_env:
-    cmd: 'source /home/jenkins/set_gcc7.5.0_env'
+    cmd:
+      16: 'source /home/jenkins/set_gcc7.5.0_env'
+      next: 'source /home/jenkins/set_gcc7.5.0_env'
```

Comment:
Why this change?

Reply:
The resources that the `set_gcc7.5.0_env` scripts setups up on static hosts is the default in the docker container.